diff --git a/cmake/vendor.cmake b/cmake/vendor.cmake
index cbdcfb47..78df374e 100644
--- a/cmake/vendor.cmake
+++ b/cmake/vendor.cmake
@@ -171,9 +171,6 @@ CPMAddPackage(
     GITHUB_REPOSITORY ocornut/imgui
     GIT_TAG v1.91.5-docking
     DOWNLOAD_ONLY TRUE
-    PATCHES
-        "${CMAKE_CURRENT_LIST_DIR}/imgui-emscripten.patch"
-        "${CMAKE_CURRENT_LIST_DIR}/imgui-loader.patch"
 )
 
 set(IMGUI_SOURCES
diff --git a/server/TracyWorker.cpp b/server/TracyWorker.cpp
index 382f2281..07b6e3ba 100644
--- a/server/TracyWorker.cpp
+++ b/server/TracyWorker.cpp
@@ -3846,7 +3846,7 @@ void Worker::AddSymbolCode( uint64_t ptr, const char* data, size_t sz )
         rval = cs_open( CS_ARCH_ARM, CS_MODE_ARM, &handle );
         break;
     case CpuArchArm64:
-        rval = cs_open( CS_ARCH_AARCH64, CS_MODE_ARM, &handle );
+        rval = cs_open( CS_ARCH_ARM64, CS_MODE_ARM, &handle );
         break;
     default:
         assert( false );
@@ -3890,9 +3890,9 @@ void Worker::AddSymbolCode( uint64_t ptr, const char* data, size_t sz )
                         }
                         break;
                     case CpuArchArm64:
-                        if( detail.aarch64.op_count == 1 && detail.aarch64.operands[0].type == AARCH64_OP_IMM )
+                        if( detail.arm64.op_count == 1 && detail.arm64.operands[0].type == ARM64_OP_IMM )
                         {
-                            callAddr = (uint64_t)detail.aarch64.operands[0].imm;
+                            callAddr = (uint64_t)detail.arm64.operands[0].imm;
                         }
                         break;
                     default:
